File: leo2-node/src/main/java/org/deku/leo2/node/data/repositories/SectorRepository.java
package org.deku.leo2.node.data.repositories;

import org.deku.leo2.node.data.entities.Sector;
import org.deku.leo2.node.data.entities.SectorPK;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.data.querydsl.QueryDslPredicateExecutor;

/**
 * Created by masc on 16.05.15.
 */
public interface SectorRepository extends JpaRepository<Sector, SectorPK>, QueryDslPredicateExecutor {
}

File: leoz-node/src/main/java/org/deku/leoz/node/data/repository/SectorRepository.kt
package org.deku.leoz.node.data.repository

import org.deku.leoz.node.data.jpa.MstSector
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.data.querydsl.QuerydslPredicateExecutor

/**
 * Sector repository
 * Created by masc on 16.05.15.
 */
interface SectorRepository :
        JpaRepository<MstSector, Long>,
        QuerydslPredicateExecutor<MstSector>
